app/models/comment.rb

```diff
@@ -3,6 +3,8 @@ class Comment < ActiveRecord::Base
   belongs_to :user
   belongs_to :application
 
+  scope :recent, -> { limit(3) }
```

Comment:
The ordering happens in the `team` model:
https://github.com/TeamCheesy/rgsoc-teams/blob/master/app/models/team.rb#L25
(One of my first commits this Summer :-) )

Shall I bring it over to the `comment` model?